crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -266,7 +259,7 @@ reveal_type(C.pure_class_variable)  # revealed: Unknown
 
 c_instance = C()
 # TODO: should be `Literal["overwritten on class"]`
-reveal_type(c_instance.pure_class_variable)  # revealed: @Todo(implicit instance attribute)
+reveal_type(c_instance.pure_class_variable)  # revealed: Unknown | Literal["value set in class method"]
```

Comment:
Probably not something that we're going to address here. This would require narrowing on attribute expressions. `Unknown | Literal["value set in class method"]` is not wrong.